apps/content/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ContentConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "apps.content"
```

Comment:
Just had a doubt. Is the `name` property here affect the imports? I mean if we change the name to something else say just `content`, would we then write `from content.models ...` instead of `from apps.content.models ...` ?

Reply:
> Full Python path to the application, e.g. 'django.contrib.admin'.
https://docs.djangoproject.com/en/4.0/ref/applications/#django.apps.AppConfig.name

`apps/__init__.py` makes `apps` directory a python module. Therefore you observed the changes correctly. We do need to import `from apps.content.models ...` however, the `AppConfig.name` just needs to reflect the path. It doesn't dictate the path.

Also note that other apps don't have this config. If we would write future proof code, we might add it. We also need to update the `INSTALLED_APPS`  values. But that is for another PR.

Reply:
Thanks for the explanation. I understand it now :)